classes/Kohana/Auth/Service.php

```diff
@@ -94,6 +94,11 @@ public function get_user()
 					if ($user->loaded())
 					{
 						$user->_is_new = FALSE;
+
+                        if (Arr::get($this->_config, 'link_user'))
+                        {
+                            $user->load_service_values($this, $data, FALSE);
+                        }
```

Comment:
I don't think we need a `link_user` configuration for exactly that as we are linking the user even when this is set to false. So we can either have it without a config variable and do it always when we link the user or rename the config variable.